web/modules/custom/joinup_licence/joinup_licence.info.yml

```diff
@@ -5,30 +5,31 @@ core: 8.x
 package: Joinup
 
 dependencies:
-  - 'field_group:field_group'
-  - 'smart_trim:smart_trim'
+  - 'drupal:taxonomy'
+  - 'joinup:joinup_validation'
   - 'spdx:spdx'
 
 config_devel:
   install:
     - core.base_field_override.rdf_entity.licence.label
-    - core.entity_form_display.rdf_entity.licence.default
-    - core.entity_view_display.rdf_entity.licence.default
-    - core.entity_view_display.rdf_entity.licence.view_mode_search_list_item
-    - core.entity_view_display.rdf_entity.licence.view_mode_tile
     - core.entity_view_mode.rdf_entity.view_mode_search_list_item
     - field.field.rdf_entity.licence.field_licence_deprecated
-    - field.field.rdf_entity.licence.field_licence_description
-    - field.field.rdf_entity.licence.field_licence_legal_type
     - field.field.rdf_entity.licence.field_licence_spdx_licence
-    - field.field.rdf_entity.licence.field_licence_type
     - field.storage.rdf_entity.field_licence_deprecated
     - field.storage.rdf_entity.field_licence_description
-    - field.storage.rdf_entity.field_licence_legal_type
     - field.storage.rdf_entity.field_licence_spdx_licence
-    - field.storage.rdf_entity.field_licence_type
     - filter.format.licence
     - rdf_entity.rdfentity.licence
     - sparql_entity_storage.mapping.rdf_entity.licence
+  optional:
+    - core.entity_form_display.rdf_entity.licence.default
+    - core.entity_view_display.rdf_entity.licence.default
+    - core.entity_view_display.rdf_entity.licence.view_mode_search_list_item
+    - core.entity_view_display.rdf_entity.licence.view_mode_tile
+    - field.field.rdf_entity.licence.field_licence_description
+    - field.field.rdf_entity.licence.field_licence_legal_type
+    - field.field.rdf_entity.licence.field_licence_type
+    - field.storage.rdf_entity.field_licence_legal_type
+    - field.storage.rdf_entity.field_licence_type
     - sparql_entity_storage.mapping.taxonomy_term.licence_type
     - taxonomy.vocabulary.licence_type
```

Comment:
As explained in a different comment, this config is no longer maintained since we converted from a distribution to a project, the config is now in `config/sync` and all old config should be removed, but this needs to be done in 2 steps since existing KernelTests still depend on it. The first step is to move everything to optional, then finally it can be removed in a followup.